include/broker/worker.hh

```diff
@@ -16,6 +17,16 @@ public:
 
   struct impl;
 
+  // -- constants --------------------------------------------------------------
+
+#ifdef BROKER_WINDOWS
```

Comment:
Explain the `*2` with a comment?

Reply:
I don't actually know the reasoning for this one. @Neverlord do remember why this is?

Reply:
Ups, sorry for the delayed response. 😅

The `*2` is just a cheap workaround for now. We use placement new to construct a `caf::actor` handle into the buffer. This type is just a wrapper around a pointer and both GCC and Clang agree that `sizeof(caf::actor) == sizeof(void*)`. However, MSVC seems to add some padding. There might be ways to tell MSVC to produce a compact struct, but since `caf::actor` is not in the Broker code base I've opted for this workaround.

I didn't bother documenting it since the whole placement-new approach is ugly and meant as a temporary solution. It was the best thing to do without changing CAF. However, the much better solution is to add a getter to `caf::actor` to retrieve the underlying pointer. I plan on adding such a getter to CAF for the next release and then update the handle implementations in Broker.